hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/AlertsEngineImpl.java

```diff
@@ -671,6 +694,21 @@ private void handleAutoResolvedTriggers() {
         }
     }
 
+    private void checkMissingStates() {
+        missingStates.stream().forEach(missingState -> {
+            MissingConditionEval eval = new MissingConditionEval(missingState.getCondition(),
+                    missingState.getPreviousTime(),
+                    System.currentTimeMillis());
+            if (eval.isMatch()) {
+                rules.removeFact(missingState);
+                missingState.setPreviousTime(System.currentTimeMillis());
+                missingState.setTime(System.currentTimeMillis());
+                rules.addFact(missingState);
+                rules.addFact(eval);
```

Comment:
I'm testing locally to add the eval in any case.
So, MissingConditionEval negatives work in a regular way with Dampening model.